src/types/auth.ts

```diff
@@ -55,7 +61,7 @@ export interface StdTx extends TxValue {
 
 /** @TODO document */
 export interface StdFee {
-    amount: Coin[];
+    amount: Coin[] | null;
```

Comment:
Could you point me to SDK code where `StdFee`'s `amount` is nullable? I'm not saying it isn't, I'm just not aware of this.

Reply:
Thank you for your reply. I am currently studying cosmos-sdk. I have not seen that cosmos-sdk has requirements for `StdFee` 's` amount`, and I consider that in some business scenarios, there may be no transaction fee.